### PR TITLE
[FEATURE] Usage Statistics Events for Profiler and DataAssistant "get_expectation_suite()" methods.

### PR DESCRIPTION
**Pull request by alexsherstinsky**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

Changes proposed in this pull request:
-
-
-

After submitting your PR, CI checks will run and @cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: great_expectations/core/usage_statistics/schemas.py [hunk 1]
```diff
@@ -516,7 +516,7 @@
     "items": {"$ref": "#/definitions/anonymized_validation"},
 }
 
-anonymized_save_or_edit_expectation_suite_payload_schema = {
+anonymized_get_or_edit_or_save_expectation_suite_payload_schema = {
     "$schema": SCHEMA,
     "title": "anonymized-save-or-edit-expectation-suite-payload",
```
Comment (anthonyburdi) [inline great_expectations/core/usage_statistics/schemas.py]: Please make this consistent with the variable name

**Review by anthonyburdi — APPROVED** [review]
LGTM!

File: tests/integration/usage_statistics/test_usage_statistics_messages.py [hunk 2]
```diff
@@ -1299,6 +1299,36 @@ def test_generate_messages_with_defaults():
             "x-forwarded-for": "00.000.00.000, 00.000.000.000",
         },
     ],
+    "profiler.result.get_expectation_suite": [
+        {
+            "event": "profiler.result.get_expectation_suite",
+            "event_payload": {
```
Comment (tannerbeam) [inline tests/integration/usage_statistics/test_usage_statistics_messages.py:1305]: just want to make sure this is representative of _all_ possible payload values. it's not required in order to infer the schema, but for my sake i think i'd like to have the examples be as such.

File: tests/integration/usage_statistics/test_usage_statistics_messages.py [hunk 3]
```diff
@@ -1299,6 +1299,36 @@ def test_generate_messages_with_defaults():
...
+            "data_context_id": "00000000-0000-0000-0000-000000000002",
+            "data_context_instance_id": "10000000-0000-0000-0000-000000000002",
+            "ge_version": "0.11.9.manual_testing",
+            "x-forwarded-for": "00.000.00.000, 00.000.000.000",
+        },
+    ],
+    "data_assistant.result.get_expectation_suite": [
+        {
+            "event": "data_assistant.result.get_expectation_suite",
+            "event_payload": {
```
Comment (tannerbeam) [inline tests/integration/usage_statistics/test_usage_statistics_messages.py:1320]: just want to make sure this is representative of _all_ possible payload values. it's not required in order to infer the schema, but for my sake i think i'd like to have the examples be as such.

**Review by tannerbeam — APPROVED** [review]
left a couple of comments regarding test events. just want to double check on that stuff because it's where i go for source-of-truth on what the events might contain. also that file is what i use for schema inference.

File: tests/integration/usage_statistics/test_usage_statistics_messages.py (hunk 2 above)
Comment (alexsherstinsky) [reply tests/integration/usage_statistics/test_usage_statistics_messages.py:1305]: @Super-Tanner Confirmed -- yes, and the new test messages verify that.  Thanks.

File: tests/integration/usage_statistics/test_usage_statistics_messages.py (hunk 3 above)
Comment (alexsherstinsky) [reply tests/integration/usage_statistics/test_usage_statistics_messages.py:1320]: @Super-Tanner Confirmed -- yes, and the new test messages verify that.  Thanks.

File: great_expectations/core/usage_statistics/schemas.py (hunk 1 above)
Comment (alexsherstinsky) [reply great_expectations/core/usage_statistics/schemas.py]: DONE!  Thanks!

**Pull request merged**